app-ios/Sources/SponsorFeature/SponsorView.swift

```diff
@@ -85,9 +85,7 @@ public struct SponsorView: View {
         WithViewStore(store) { viewStore in
             ZStack(alignment: .center) {
                 if viewStore.isLoading {
-                    ProgressView()
-                        .progressViewStyle(CircularProgressViewStyle(tint: AssetColors.onBackground.swiftUIColor))
```

Comment:
`.progressViewStyle(CircularProgressViewStyle(tint: AssetColors.onBackground.swiftUIColor))`

This modifier is useful for combining with background color 🙏🏼

Reply:
I fixed `LoadingView` style at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/15d19ebaac148b557555bc9f3b173aac5e895718 !